src/pudl/transform/ferc714.py

```diff
@@ -175,27 +175,45 @@
 """Mapping between standardized time offset codes and canonical timezones."""
 
 EIA_CODE_FIXES = {
```

Comment:
instead of converting all of these code fixes into the pudl-derived respondent id-based fixes, I kept the sourcey-ness. This is mostly to enable checking things at each stage - primarily `ensure_eia_code_uniqueness`